oak_remote_attestation/proto/v1/messages.proto

```diff
@@ -49,10 +51,14 @@ message AttestationEvidence {
 // in response to its request for the enclave's public key(s).
 message AttestationBundle {
   // Attestation evidence from the enclave.
-  AttestationEvidence attestation_evidence = 1;
+  // TODO(#4488): Remove once this is no longer used.
+  AttestationEvidence attestation_evidence = 1 [deprecated = true];
 
   // Supporting evidence required for verifying the integrity of attestation evidence.
   AttestationEndorsement attestation_endorsement = 2;
+
+  // The DICE attestation evidence.
+  oak.attestation.v1.Evidence dice_evidence = 3;
```

Comment:
I think instead of adding this to the `AttestationBundle` we should add evidence and endorsements directly to the  `GetPublicKeyResponse`: https://github.com/project-oak/oak/blob/5505c52e9c3483ca8255646f8595bbe27a8cdfe9/oak_remote_attestation/proto/v1/messages.proto#L85-L88

Or will it be added there in the future?

Reply:
It can be added there in future, but at the moment the attestation bundle itself is passed around in many places. If this isn't part of the attestation bundle we would have to create a new wrapper struct that contains the attestation bundle and the DICE evidence and replace AttestationBundle with this new wrapper throughout the codebase.

Reply:
I tried adding this wrapper struct here: https://github.com/project-oak/oak/pull/4473

But I think most of the people decided to use separate structs. But I can re-open the PR and add the `EndorsedEvidence` (since it's a correct term rather than `AttestationBundle`)

Reply:
I am not clear on what you are proposing. Do you mean you want to add EndorsedEvidence to AttestationBundle? That would require a very large change through the code base. I am trying to do this in phases, starting with adding the DICE evidence to the attestation bundle. When everything uses the DICE evidence instead of the deprecated AttestationEvidence the next step is to deprecate the endorsement and replace it with the new endoresement struct. Once that is done and the public keys removed, we can rename AttestationBundle to EndorsedEvidence.

Reply:
Ah, got it

So it's a temporary change and it will become an `EndorsedEvidence` in the future

Reply:
Yes, that is the plan.

Reply:
But we shouldn't forget to rename `oak.attestation.v1.Evidence dice_evidence` to `oak.attestation.v1.Evidence evidence` for consistency

Reply:
Thanks. I have added a comment to #4488 to track that.

Reply:
Also, in the future we'll just make `Evidence` contain a `oneof {DiceEvidence, RtmrEvidence}`. So we won't have to change the `EndorsedEvidence {Evidence evidence = 1, Endorsements endorsements = 2}` message